coalib/processes/BearRunning.py

```diff
@@ -383,7 +383,7 @@ def run_local_bears(filename_queue,
     :param timeout:           The queue blocks at most timeout seconds for a
                               free slot to execute the put operation on. After
                               the timeout it returns queue Full exception.
-    :param file_dict:         Dictionary that contains contents of files.
+    :param file_dict:         Dictionary that contains fileproxy objects.
```

Comment:
So, why is this a dictionary ? FileProxy already has the filename right ? Double info is bad.
Earlier when it was just list of lines it made sense,